Functions.Templates/Bindings/bindings.json

```diff
@@ -351,26 +351,6 @@
           "help": "$httpTrigger_route_help",
           "validators": []
         },
-        {
-          "name": "webHookType",

```

Comment:
Need to remove this as well.
https://github.com/Azure/azure-functions-templates/blob/34a7327e12b7c74a79567052f9d1a48cb91fe664/Functions.Templates/Bindings/bindings.json#L426-L450

Reply:
Done